### DISCUSSION TITLE
[Feature] Manually add people

### DISCUSSION THREAD

**Discussion by MacBadger** [immich-app/immich#9853] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Sometimes, people are not detected.
As far as I can tell, you can't manual add a face. The edit feature only allow reassigning as a different person. 
This would be great to be able to do. (equally just removing a tag of an erroneously identified 'person' but this is less important)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by schuhbacca** [comment]
Duplicate of https://github.com/immich-app/immich/discussions/2472#discussioncomment-6850486

**Comment by viki53** [comment]
Should this be re-opened, with #2472 being closed but this doesn't seem to have been implemented yet?

I have people/faces not being detected in some photos and I would like to be able to tag them and add them at the same time.

**Reply by schuhbacca** [reply]
You can do this via the web interface

**Reply by viki53** [reply]
I cannot, no. People who haven't been detected yet cannot be manually added. I've tried playing with the face detection settings and it doesn't help.

I would like to be able to add a person when I add a tag on a photo. That option doesn't seem to exist with the current version (1.138).

**Reply by Ifglimir** [reply]
This is something I am also missing. Would be glad if this would be implemented.

**Reply by Sharparam** [reply]
@schuhbacca Since you seem so sure that this can be done in the web interface, please point us to where we can do this. I've looked in the "People" section but can't find anywhere to do it.

Edit: Ok I found a workaround if you have at least 1 face already recognized and available to use:
 1. Go to a photo that has the missing person you want to add.
 2. Open the info panel and then use the "+" under people
 3. Use the selection box to mark the face you want to add
 4. Add this face to one of your **existing** people
 5. Go to the person you added it to
 6. Select the photo you used
 7. Go to the triple dot menu and then "Fix incorrect match"
 8. Choose to create a new person

Doing it this way is just a workaround though and a proper way still should be added, I'm also not sure if this aids the machine learning to recognize future faces of the new person.

Edit 2: So found a better way in the case where Immich's facial recognition has actually recognized a face in the photo but not created a person for it under the "People" menu. If you go to the people section on a photo and choose to edit instead, you'll see the auto-detected faces there and can go "edit" on one of them and then use the "+" to create a new person from there. I'm guessing this is maybe what @schuhbacca was referring to?
